aws/sdk/build.gradle.kts

```diff
@@ -260,10 +200,9 @@ tasks.register("relocateAwsRuntime") {
     dependsOn("copyAllRuntimes")
     doLast {
         // Patch the Cargo.toml files
-        awsModules.forEach { moduleName ->
+        Crates.AWS_RUNTIME.forEach { moduleName ->
```

Comment:
might be worth thinking of a different naming scheme here, my brain doesn't expect this to be a list.